projects/plugins/jetpack/class.jetpack.php

```diff
@@ -2033,15 +2033,21 @@ public static function load_modules() {
 			$modules = array_diff( $modules, $updated_modules );
 		}
 
+		$is_userless = self::connection()->is_userless();
```

Comment:
The meaning behind the check LGTM, but I can't help but say that using internal verbiage is not my favorite way to name things. It does not hold its meaning over time, nor does it mean anything to developers who aren't in the know.  A better name for this would be something like `is_site_connection_only` or `no_connected_users`. But this is not a blocker.

Reply:
I totally agree and discussed that with @zinigor and @fgiannar before. I think this should be changed in a following PR. Let's do this @fgiannar ?